x/tokenfactory/types/params.go

```diff
@@ -30,8 +31,9 @@ func NewParams(denomCreationFee sdk.Coins, denomCreationGasConsume uint64) Param
 // default gamm module parameters.
 func DefaultParams() Params {
 	return Params{
-		DenomCreationFee:        sdk.NewCoins(sdk.NewInt64Coin(appparams.BaseCoinUnit, 10_000_000)), // 10 OSMO
-		DenomCreationGasConsume: 0,
+		// For choice, see: https://github.com/osmosis-labs/osmosis/pull/4983
+		DenomCreationFee:        sdk.NewCoins(), // used to be 10 OSMO at launch.
+		DenomCreationGasConsume: uint64(DefaultCreationGasFee),
```

Comment:
See discussion for context: https://osmosis-network.slack.com/archives/C04LZFBHGQ5/p1685588880275719?thread_ts=1685506174.651869&cid=C04LZFBHGQ5

Also, see this: https://github.com/osmosis-labs/osmosis/pull/4983